src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -58,23 +58,9 @@ private async static Task<IQueryable<IPackageSearchMetadata>> execute_package_se
             searchFilter.IncludeDelisted = configuration.ListCommand.LocalOnly;
             var cacheContext = new ChocolateySourceCacheContext(configuration);
 
-            /*
-            // Whether or not the package is remote determines two things:
-            // 1. Does the repository have a notion of "listed"?
-            // 2. Does it support prerelease in a straight-forward way?
-            // Choco previously dealt with this by taking the path of least resistance and manually filtering out and sort unwanted packages
-            // This result in blocking operations that didn't let service based repositories, like OData, take care of heavy lifting on the server.
-            bool isServiceBased;
-            var aggregateRepo = packageRepository as AggregateRepository;
-            if (aggregateRepo != null)
-            {
-                isServiceBased = aggregateRepo.Repositories.All(repo => repo is IServiceBasedRepository);
-            }
-            else
-            {
-                isServiceBased = packageRepository is IServiceBasedRepository;
-            }
-            */
+
+            //TODO, maybe adjust number if no page specified? Or adjust check to have no max value

```

Comment:
Has an issue/task been raised to track this todo?

Reply:
https://app.clickup.com/t/20540031/PROJ-453